Detectors/MUON/MCH/Raw/Decoder/src/ROFFinder.cxx

```diff
@@ -145,10 +145,12 @@ void ROFFinder::sortDigits()
   for (size_t i = 0; i < mInputDigits.size(); i++) {
     auto& digit = mInputDigits[i];
     if (!digit.timeValid()) {
+#ifdef ROFDEBUG
       LOG(ERROR) << "Digit with invalid time, DS " << digit.info.solar << "," << digit.info.ds << "," << digit.info.chip
```

Comment:
note that FairLogger is deprecating the upper case version of the log levels, so from now on, we should use the lower case versions when we think about it